samples/README.md

```diff
@@ -25,43 +49,96 @@ exporters:
     - name: LOG_LEVEL
       value: DEBUG
     - name: NAMESPACES
-      value: basic-nginx-prod
+      value: mongo-persistent
     source_ref: master
-    source_url: https://github.com/konveyor/pelorus.git
+    source_url: https://github.com/your_org/pelorus.git
   - app_name: committime-generic
     env_from_secrets:
     - github-secret
-    - mongodb-build-webhook
     source_context_dir: exporters/
     extraEnv:
     - name: APP_FILE
       value: committime/app.py
     - name: LOG_LEVEL
       value: DEBUG
     source_ref: master
-    source_url: https://github.com/konveyor/pelorus.git
+    source_url: https://github.com/your_org/pelorus.git
+```
+3. Documentation regarding values.yaml can be found [here](https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md).  
+  Apply the updated values for Pelorus by executing:
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+***Note*** pause to allow the committime exporter pod to build and deploy.
+
+Pelorus should now be installed and configured to measure the todolist sample app. We'll have to deploy the sample application to view measurements from Pelorus's Grafana dashboard.
+
+## Todolist sample application install steps
+
+1. git clone the forked copy of the 
+```
+git clone https://github.com/your_org/mig-demo-apps.git
```

Comment:
Can we simply do:
```
export GITHUB_ORG=<your_org>
git clone "https://github.com/${GITHUB_ORG}/mig-demo-apps.git"
```


and then in the following step (number 2):

`sed -i "s/your_org/${GITHUB_ORG}/g" mongo-persistent.yaml
`

**NOTE the double quotes around sed expression, it will NOT work with single quote.**